src/test/java/org/openmrs/contrib/qaframework/test/AddRolesToUserTest.java

```diff
@@ -0,0 +1,86 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ * 
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.contrib.qaframework.test;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Random;
+
+import org.junit.After;
+import org.junit.Ignore;
+import org.openmrs.contrib.qaframework.helper.ReferenceApplicationTestBase;
+import org.openmrs.contrib.qaframework.page.AdministrationPage;
+import org.openmrs.contrib.qaframework.page.ManageUserPage;
+
+@Ignore
+public class AddRolesToUserTest extends ReferenceApplicationTestBase {
```

Comment:
why adding a test that is ignored to run with ci? AddRolesToUserTest seems to have not been constructed well during its development and I think better not to add it since we have [UserAccount](https://github.com/openmrs/openmrs-contrib-qaframework/blob/master/src/test/java/org/openmrs/contrib/qaframework/automation/UserAccountSteps.java) workflow test that covers adding roles to users, checkout how roles are added to [users ](https://github.com/openmrs/openmrs-contrib-qaframework/blob/6b5c48d722e09c0ab46f1dfd2a160d244ec2642d/src/test/java/org/openmrs/contrib/qaframework/automation/UserAccountSteps.java#L85-L88)